pkg/values/packagemaps.go

```diff
@@ -58,6 +58,148 @@ type ModelPackageMap map[DistroFamilyInterface]map[Architecture]map[Model]Versio
 // VersionMap is a map of a constraint to a list of packages
 type VersionMap map[string][]string
 
+// BasePackagesModels allows model-specific base packages per distro/arch/model
+var BasePackagesModels = ModelPackageMap{
```

Comment:
@Itxaka I added these to follow a similar approach to KernelPackagesModels, but I don't understand why we have to distinguish from Generic, IMO it would be better to be able already in KernelPackages, or BasePackages to have this granularity